tests_speculos/test-transactions.js

```diff
@@ -207,29 +143,25 @@ const exampleAddKeyBlob        = "f90186f9015eb86e7472616e73616374696f6e28707562
 		{
 			title: "Transfer FLOW",
 			blob: exampleTransferBlob,
-			pageCount: (process.env.TEST_DEVICE && process.env.TEST_DEVICE == "nanox")?12:12,
```

Comment:
Are we no longer testing page counts?

Reply:
We do not count the pages, but the screenshots have to match (including ordering). On change, you have to carefully review the screenshots content. Thus I do not think we really decrease test coverage. Very often, I needed snapshots to validate / correct these numbers. 
This will save quite some time writing and modifying the tests.